### PR TITLE
Creating message body

### PR DESCRIPTION
**Pull request by josedanielasn**
1. Created new table for **message** with columns: "message_body" "user_id". This belongs to **User** table.
2. Created Controller for creating message (messages_controller) and for displaying (chatrooms_controller).
3. Created ERB(HTML) for displaying and Creating messages.
4. Added design in ERB(HTML) for login, sign up,  and displaying and creating messages using bootstrap.

Screen Shot 2020-09-18 at 10 06 59 AM
Screen Shot 2020-09-17 at 5 41 40 PM
Screen Shot 2020-09-17 at 5 41 29 PM

### CODE REVIEW

File: app/controllers/chatrooms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class ChatroomsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @user = User.find(params[:user_id])
```
Comment (analyn-cajocson) [inline app/controllers/chatrooms_controller.rb]: you don't have to do this manually. check the documentation of `devise` for getting the current logged in user.

File: app/controllers/messages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class MessagesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @messages = Message.new(messages_params)
+    @messages.user_id = params[:user_id]
```
Comment (analyn-cajocson) [inline app/controllers/messages_controller.rb]: refer to the comment above

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
-  get 'pages/home'
+  root to: 'pages#home'
```
Comment (analyn-cajocson) [inline config/routes.rb]: redirect the root page to `chatroom/index`

File: config/webpack/environment.js [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
 
 module.exports = environment
+
+const {
+  environment
+} = require('@rails/webpacker')
+
+const {
+  environment
+} = require('@rails/webpacker')
```
Comment (analyn-cajocson) [inline config/webpack/environment.js]: Duplicate code.... I guess?

File: config/webpack/environment.js [hunk 5]
```diff
@@ -1,3 +1,25 @@
-const { environment } = require('@rails/webpacker')
+const {
+  environment
+} = require('@rails/webpacker')
 
 module.exports = environment
```
Comment (analyn-cajocson) [inline config/webpack/environment.js]: Is it necessary to put this code at the end of this file?

Comment (josedanielasn) [reply config/webpack/environment.js]: Ma'am na remove ko na po

Comment (analyn-cajocson) [reply config/webpack/environment.js]: I'm talking about the `module.exports = environment`. Do we need to put this at the end of this file? I saw some resources that say it should be at the end of this file. What do you think?

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**